benchmarks/src/jmh/java/io/micronaut/http/server/stack/SearchController.java

```diff
@@ -0,0 +1,39 @@
+package io.micronaut.http.server.stack;
+
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.core.annotation.Introspected;
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.MutableHttpResponse;
+import io.micronaut.http.annotation.Body;
+import io.micronaut.http.annotation.Controller;
+import io.micronaut.http.annotation.Post;
+
+import java.util.List;
+
+@Controller("/search")
+@Requires(property = "spec.name", value = "FullHttpStackBenchmark")
+public class SearchController {
+    @Post("find")
+    public HttpResponse<?> find(@Body Input input) {
```

Comment:
It would be nice to test more variations, like returning a simple string, a completable future reactive response.
Right now the async flow (completable future) is executed using the reactor which can be improved

Reply:
this benchmark is from micronaut-benchmark, which does not use reactive / async code so that it's a more fair comparison between frameworks. benchmarking reactive code is also worthwhile but this simple controller is basically the best "bare-bones" test of the http stack

Reply:
Ok, having some other benchmark testing overhead of async/reactive flow would be nice